specs/reservation_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'spec_helper'
+
+describe "Reservation class" do
+
+  before do
+    start_date = Date.new(2018,5,20)
```

Comment:
In general, try to surround blocks with whitespace on either side for readability. Think of them as paragraphs:

```ruby
# no
    start_date = Date.new(2018,5,20)
    end_date = start_date + 4
    @reservation_data = {
      res_id: 8,
      room: Hotel::Room.new(room_number: 3),
      start_date: start_date,
      end_date: end_date
    }
    @reservation = Hotel::Reservation.new(@reservation_data)

# yes
    start_date = Date.new(2018,5,20)
    end_date = start_date + 4

    @reservation_data = {
      res_id: 8,
      room: Hotel::Room.new(room_number: 3),
      start_date: start_date,
      end_date: end_date
    }

    @reservation = Hotel::Reservation.new(@reservation_data)
```